Source/Utils/WarpXUtil.cpp

```diff
@@ -376,8 +376,10 @@ parseStringtoInt(std::string str, std::string name)
     return ival;
 }
 
+// overloads for float/double instead of amrex::Real to allow makeParser()
+// to query for my_constants as double even in single precision mode
```

Comment:
Perhaps also add "Always parsing in double precision avoids potential overflows that may occur when parsing user's expressions because of the limited range of exponentials in single precision."